README.md

```diff
@@ -1 +1,256 @@
-# 1st-Capstone-Project
\ No newline at end of file
+# HTML, CSS & JavaScript Capstone Project
+HTML, CSS and JavaScript Project
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [🎥 Video Explanation](#video-explanation)   
+- [🚀 Live Demo](#live-demo)    
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+-- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+- [© Attribution](#attribution)
+
+# 📖 [Capstone_Project] <a name="Capstone Project"></a>
+
+
+**[Capstone_Project]** is a simple HTML, CSS and JavaScript project that uses a simple web design to create a UI in both mobile and desktop version, use JavaScript events and DOM elements to create pages dynamically. 
+
+## 🛠 Built With  <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details>
+
+<!-- VIdeo Explanation -->
+## 🎥 Video Explanation <a name="video-explanation"></a>
+
```

Comment:
- [ ] Kindly, make sure you added some or show some lines of code in your video 👍